webui/react/src/routes/routes.ts

```diff
@@ -47,12 +47,11 @@ const routes: RouteConfig[] = [
     title: 'Uncategorized',
   },
   {
-    icon: 'experiment',
-    id: 'uncategorized',
+    id: 'workspaceList',
     needAuth: true,
     path: '/',
-    redirect: '/projects/1',
-    title: 'Uncategorized',
+    redirect: '/workspaces',
+    title: 'Workspace',
```

Comment:
Does this change the redirect to be the workspaces list for everybody?

Reply:
Yes, should we only redirect to workspace list when rbac is on?

Reply:
Yes, let's condition that on the rbac feature switch.

Reply:
No problem